package.json

```diff
@@ -5,11 +5,12 @@
   "author": "Qiskit Development Team",
   "license": "Apache-2.0",
   "scripts": {
-    "check": "npm run check:metadata && npm run check:spelling && npm run check:links && npm run check:fmt",
+    "check": "npm run check:metadata && npm run check:spelling && npm run check-internal-links && npm run check:fmt",
     "check:metadata": "node -r esbuild-register scripts/commands/checkMetadata.ts",
-    "check:links": "node -r esbuild-register scripts/commands/checkLinks.ts",
     "check:spelling": "cspell --relative --no-progress docs/**/*.md* docs/api/**/*.md* --config cspell/cSpell.json",
     "check:fmt": "prettier --check .",
+    "check-internal-links": "node -r esbuild-register scripts/commands/checkInternalLinks.ts",
+    "check-external-links": "node -r esbuild-register scripts/commands/checkExternalLinks.ts",
     "check-pages-render": "node -r esbuild-register scripts/commands/checkPagesRender.ts",
```

Comment:
Nit: What do you think about changing these three commands to follow the `check:` convention? I think it would be good given that they start with the word `check`, and we also have `check:metadata`, `check:spelling,` and `check:fmt`